src/vm/fastserializableobject.h

```diff
@@ -25,6 +25,14 @@ class FastSerializableObject
 
     // Get the type name for the current object.
     virtual const char* GetTypeName() = 0;
+
+    int GetObjectVersion() const { return m_objectVersion; }
```

Comment:
For these Get functions, can you add LIMITED_METHOD_CONTRACT contract?  See https://github.com/dotnet/coreclr/blob/master/src/vm/eventpipeconfiguration.cpp#L210